GeeksCoreLibrary/Modules/PostalServices/PostNL/Models/AddressModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace GeeksCoreLibrary.Modules.PostalServices.PostNL.Models
+{
+    public class AddressModel
+    {
+        public string CompanyName { get; set; }
+        public string AddressType { get; set; }
+        public string City { get; set; }
+        public string Countrycode { get; set; }
+        public string FirstName { get; set; }
+        public string HouseNr { get; set; }
+        public string HouseNrExt { get; set; }
```

Comment:
Geen afkortingen. Hernoemen naar `HouseNumberExtension` of `HouseNumberAddition`.